### PR TITLE
"Informative" Image Selection & Form Recognizer Extraction Pattern

### PR DESCRIPTION
**Pull request by dafatade**
This PR focuses on an Informative Image Selection and Form Recognizer Extraction Pattern, where "informative" image selection is used as a preprocessing mechanism to weed out redundant and potentially less useful data points. The results from this selection process are then fed into a custom Form Recognizer service where predictions are computed. The PR contains:

- MLOps utility functions
- Utility functions for the Form Recognizer and OCR services
- A sample Custom Form Recognizer Training pipeline
- A scoring pipeline that utilizes "informative" image selection and the trained custom Form Recognizer service to produce a set of key-value pairs using metadata extracted from clapperboard images
- Documentation regarding Pipelines and utility functions
- Sample Training and Validation data

### CODE REVIEW

File: Scenarios/Informative_Image_Selection_FR_Pattern/docs/how_to_setup.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+In order to train a custom Form Recognizer service, the following assets are needed for each image file:
+
+- A `labels.json` file
+- An `ocr.json` file
+
+Currently, a set of asset files used to train the Custom Form Recognizer service can be found under the `data/clapperboard_asset_files` directory. This directory contains the training images as well as the label and OCR files generated for each image using the sample labeling tool. There are also just the plain set of clapperboard images used for training in the `images/train/clapperboard` directory in the event that you wish to define a different set of labels. Feel free to use a completely different dataset! Just make sure the asset files required for training on that new dataset are provisioned using the sample labeling tool and uploaded to a blob storage container.
```
Comment (shanepeckham) [inline Scenarios/Informative_Image_Selection_FR_Pattern/docs/how_to_setup.md:23]: Can we make `data/clapperboard_asset_files` and `images/train/clapperboard` links instead

File: Scenarios/Informative_Image_Selection_FR_Pattern/docs/how_to_setup.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+The pipelines utilized within this experiment make use of the following cognitive service:
+
+- Form Recognizer service
+- Read API service (under Azure Computer Vision service)
+
+In order to successfully run any experiments, both services need to be provisioned. These services can be accessed and created through the Azure portal. Once a subscription for both the Form Recognizer and Computer Vision service have been created, the credentials should be stored for use in the next step (step #4).
+
+## Step 4: Add Secrets as Environment Variables ##
+
+Now that everything is in place, we simply need to set up our environment variables in order to run the sample notebooks. In the `demo` directory the following notebooks are available:
```
Comment (shanepeckham) [inline Scenarios/Informative_Image_Selection_FR_Pattern/docs/how_to_setup.md:69]: Add `demo` as link

File: Scenarios/Informative_Image_Selection_FR_Pattern/.gitignore [hunk 3]
```diff
@@ -0,0 +1,119 @@
+# Byte-compiled / optimized / DLL files
```
Comment (shanepeckham) [inline Scenarios/Informative_Image_Selection_FR_Pattern/.gitignore:1]: Can you add a requirements.txt file for this project and keep it in the root of this solution folder

Comment (shanepeckham) [reply Scenarios/Informative_Image_Selection_FR_Pattern/.gitignore:1]: Can you also please add a root README entry to this solution in the Example Scenarios section

Comment (shanepeckham) [reply Scenarios/Informative_Image_Selection_FR_Pattern/.gitignore:1]: Also add yourself to the Contributors section in the root README

**Review by shanepeckham — CHANGES_REQUESTED** [review]

File: Scenarios/Informative_Image_Selection_FR_Pattern/docs/how_to_setup.md (hunk 1 above)
Comment (dafatade) [reply Scenarios/Informative_Image_Selection_FR_Pattern/docs/how_to_setup.md]: Added links :)

**Review by shanepeckham — APPROVED** [review]

**Pull request merged**